include/deal.II/distributed/tria_base.h

```diff
@@ -239,6 +239,10 @@ namespace parallel
     fill_level_ghost_owners();
   };
 
+
```

Comment:
Can you please add something like:
```
/**
 * Using directive for backwards-compatibility.
 * @deprecated Use TriangulationBase instead of Triangulation.
 */
```

Reply:
Done!